ts/backend/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "description": "FAB backend",
   "scripts": {
-    "start": "npm run private:serve & npm run private:test",
+    "start": "concurrently --kill-others 'npm run private:serve' 'npm run private:test'",
```

Comment:
Add to  [documentation](https://github.com/flatland-association/flatland-benchmarks/blob/7c12357926d4f53158c6d009ff2dc21fe94b9566/ts/backend/README.md?plain=1#L32-L47) that tests are also run?